gis/grass/Portfile

```diff
@@ -108,7 +102,9 @@ configure.args-append \
                     --with-zstd-libs=${prefix}/lib \
                     --with-x=no
 
-set python_suffixes {39 310 311 312}
+set python_suffixes {39 310 311 312 313}
+# keep python_default in sync with qgis3
+set python_default 312
```

Comment:
this should be Python 3.13 now if possible as that is MacPort's default

Reply:
qgis3 is not ready for 3.13 (indirectly missing py-numba), and users of QGIS may install with +grass variant, so it is preferable if they are in sync.